### PR TITLE
kafka-python: intial integration

### PR DESCRIPTION
**Pull request by arthurscchan**
Create fuzzers

### CODE REVIEW

File: projects/kafka/build.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Build and install project (using current CFLAGS, CXXFLAGS).
+pip3 install --upgrade pip
```
Comment (jonathanmetzman) [inline projects/kafka/build.sh:19]: Why not do this in dockerfile?

Comment (arthurscchan) [reply projects/kafka/build.sh]: Moved to Dockerfile

File: projects/kafka/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder-python
+
+RUN apt-get update -y
+RUN apt-get install wget software-properties-common lsb-release gcc make python3-dev libsasl2-modules-gssapi-mit krb5-user -y
```
[the remark this answers is not in the document]
Comment (arthurscchan) [reply projects/kafka/Dockerfile]: Rearranged those apt-get commands

[the remark this answers is not in the document]
Comment (arthurscchan) [reply projects/kafka/Dockerfile]: Thanks for the suggestions.

File: projects/kafka/build.sh (hunk 1 above)
Comment (arthurscchan) [reply projects/kafka/build.sh]: Thanks for the suggestions.

**Pull request merged**